source/Forthon_options.py

```diff
@@ -101,6 +101,11 @@
 parser.add_option('--writemodules', action='store_true', default=True, dest='writemodules')
 parser.add_option('--nowritemodules', action='store_false', default=True, dest='writemodules', help="Don't write out the module definitions. Useful if the modules have been written already. Note that if variables of derived type are used, the original code will need to be modified. See example2. Also note that if this option is used, no checks are made to ensure the consistency between the interface file description and the actual module.")
 
+# OMP related options
```

Comment:
I have the options listed in alphabetical order. Can you move these to between '--macro' and '--pkgbase'?